api/v1/controllers/botData.js

```diff
@@ -13,12 +13,14 @@
 
 const helper = require('../helpers/nouns/botData');
 const Bot = require('../../../db').Bot;
+const BotData = require('../../../db').BotData;
 const doPost = require('../helpers/verbs/doPost');
 const doFind = require('../helpers/verbs/doFind');
 const doGet = require('../helpers/verbs/doGet');
 const doPatch = require('../helpers/verbs/doPatch');
 const doDelete = require('../helpers/verbs/doDelete');
 const u = require('../../../utils/common');
+const bdUtils = require('../../../db/helpers/botDataUtils');
```

Comment:
I don't *love* calling into a db helper module from api layer.
That being said, I don't want you to have to jump through hoops to move stuff and I don't want you to duplicate code.
Use your best judgement :)

Reply:
Its called "botDataUtils" i think it feels okay, if i have to use the file again ill probably move it to utils more directly accessible